lib/smart_app_launch/metadata.rb

```diff
@@ -30,16 +30,19 @@ class Metadata < Inferno::TestKit
 
       ## Status
 
-      The SMART App Launch Test Kit primarily verifies that systems correctly
-      implement the SMART App Launch IG for providing authorization and/or
-      authentication services to client applications accessing HL7 FHIR APIs.
-
-      The test kit currently tests the following requirements:
-      - Standalone Launch
-      - EHR Launch
-
-      It also tests the ability of a Brand Bundle Publisher to publish a valid brand
-      bundle as described in the User-access Brands and Endpoints specification.
+      The SMART App Launch Test Kit provides five suites that verify that systems
+      correctly implement different aspects and versions of the SMART App Launch IG.
+      - Three server suites (SMART App Launch STU{1, 2, 2.2}) verifying that SMART server implementations
```

Comment:
> SMART App Launch STU{1, 2, 2.2}

Is this some sort of standard nomenclature that we're using? I'm not sure why it's necessary when `STU2 and 2.2` is used below.

Reply:
I'm not quite sure what you're referring to, but I tried to adjust. Please make a more concrete suggestion if this isn't good.

Another question, should we update all of these titles to separate "STU" and the number? (e.g., SMART App Launch STU 2.2 instead of STU2.2?)

Reply:
I was referring to `STU{1, 2, 2.2}`, which is a format I don't think I've seen elsewhere.

It looks like FHIR isn't consistent with STU # vs. STU#, so I don't have strong feelings.

![Screenshot 2025-04-22 at 7 58 51 AM](https://github.com/user-attachments/assets/ce7ed684-746a-456e-a985-882b40131974)

Reply:
Let's not complicate things and leave it as is. Are you happy with the update I made?